packages/react/src/field/Field.css.ts

```diff
@@ -0,0 +1,8 @@
+import { recipe, style } from "../vanilla-extract";
+
+export const icon = recipe({
+  base: style({
+    height: "12px",
+    width: "14px",
```

Comment:
we don't need a css.ts file here - cause you can just do `h="12" w="auto"` to achieve the same thing in sprinkles